### DISCUSSION TITLE
How to specify the correct case for the hreflang attribute?

### DISCUSSION THREAD

**Discussion by kliatskiidenis** [vercel/next.js#69716] [App Router]
Greetings
I am developing on Nextjs 14 with app routing, I need to write an alternate link for SEO, I do it as indicated in the official documentation, but I get hrefLang instead of the required hreflang (in lower case)

<img width="692" alt="Знімок екрана 2024-09-05 о 11 42 23" src="https://github.com/user-attachments/assets/8f6aa6b8-d560-4bbf-8863-7d058a6fc609">
<img width="475" alt="Знімок екрана 2024-09-05 о 11 34 29" src="https://github.com/user-attachments/assets/f0022b86-53aa-450c-b790-e6f5da0870a0">
Please help. I have seen similar questions, but there is no answer. What should be done to make it work as it should? this is not an error and everything will work as it should? 
Is it possible to put it into production?

**Comment by Juneezee** [comment]
There is an open PR that fixes this https://github.com/vercel/next.js/pull/66983, but it is not merged yet.

> this is not an error and everything will work as it should?

Yes, `hrefLang` should work as it is like `hreflang`. According to the HTML spec [^1]:

> In the HTML syntax, attribute names, even those for [foreign elements](https://html.spec.whatwg.org/multipage/syntax.html#foreign-elements), may be written with any mix of [ASCII lower](https://infra.spec.whatwg.org/#ascii-lower-alpha) and [ASCII upper alphas](https://infra.spec.whatwg.org/#ascii-upper-alpha).

[^1]: https://html.spec.whatwg.org/multipage/syntax.html#attributes-2

**Comment by amit-kumar-fynd-external** [comment]
when this PR is going to merge. It is causing the problem in SEO.

**Comment by kliatskiidenis** [comment]
My SEO department also says that they do not fit hrefLang should be hreflang
Is there any solution?
